app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -66,9 +66,12 @@ def ems_inv_to_hashes
       # get_hosts
       get_images
       get_servers
+      # TODO: change spec to remove this
+      #
       get_volumes
```

Comment:
I don't think you need to worry about Travis failures for WIP PRs.  And I'm not sure why leaving Swift in Openstack and not pulling it out into separate files causes Travis failures.